python/setup.py

```diff
@@ -17,6 +17,8 @@
 
 REQUIREMENTS = [
     "numpy",
+    "scipy",
+    "pybind11",
```

Comment:
This is tricky - `pip install pybind11` isn't enough because it misses the `pybind11` CMake files.